src/app/design/frontend/Scandiweb/pwa/src/app/component/ContactExpandableContent/ContactExpandableContent.style.scss

```
@@ -0,0 +1,213 @@
+   }
+  }
+}
+
+.Form {
+  &-ContactExpandableContentHeading {
+    &::before{
+    content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='13' height='13' viewBox='0 0 18 17'%3E%3Cg%3E%3Cpath d='M.019 16.37c-.05.171 0 .36.126.485.093.092.222.145.351.145.047 0 .093-.007.136-.02l4.634-1.316-3.924-3.907L.019 16.37z' transform='translate(.375)'/%3E%3Cpath d='M.515 17c-.137 0-.267-.053-.364-.15-.133-.132-.183-.325-.13-.504l1.343-4.668 3.995 3.97L.656 16.98c-.047.013-.094.02-.14.02zm.862-5.263L.051 16.353c-.047.169 0 .348.124.471.09.09.213.14.34.14.044 0 .09-.007.134-.02l4.643-1.319-3.915-3.888zM16.567 2.55L14.452.44C14.166.156 13.792 0 13.39 0c-.4 0-.776.156-1.06.436l-1.058 1.057 4.233 4.227 1.059-1.058c.283-.283.436-.658.436-1.057.003-.4-.153-.772-.433-1.054z' transform='translate(.375)'/%3E%3Cpath d='M15.501 5.72l-4.23-4.22L12.335.44c.285-.285.662-.44 1.062-.44.404 0 .781.155 1.063.44l2.101 2.096c.284.284.44.66.44 1.06 0 .403-.156.78-.44 1.06L15.501 5.72zM11.318 1.5l4.183 4.173 1.04-1.037c.277-.277.43-.644.43-1.036 0-.393-.153-.76-.43-1.037l-2.102-2.1c-.278-.278-.649-.43-1.043-.43-.393 0-.76.152-1.039.43l-1.039 1.036zM1.94 10.872L10.57 2.224 14.783 6.446 6.153 15.093z' transform='translate(.375)'/%3E%3Cpath d='M6.1 15.093l-4.252-4.257 8.685-8.691 4.25 4.254L6.1 15.093zm-4.202-4.257L6.1 15.044l8.635-8.645-4.203-4.207-8.635 8.644z' transform='translate(.375)'/%3E%3C/g%3E%3C/svg%3E%0A");
+   }
+  }
+}
+
+.Social {
+  &-ContactExpandableContentContent {
```

And Here you will have 
```suggestion
  &-ContactExpandableContent {
      &-Content {}
      &-Heading {}
```